src/main.js

```diff
@@ -0,0 +1,315 @@
+    }
+
+    project.save(projects);
+    return saveLastProject(project);
+  };
+
+  // renders todo form
+
+  const renderForm = (project = null, toDo = null) => {
+    const validateForm = document.getElementById('todo-form');
+    if (validateForm) {
+      validateForm.parentNode.removeChild(validateForm);
+      return false;
+    }
+
+    Form.render();
+
+    const form = document.getElementById('todo-form');
+
+    const submit = document.createElement('button');
+    submit.setAttribute('type', 'submit');
+    submit.classList.add('form-button');
+    submit.onclick = () => validateToDo(project, toDo);
+    if (toDo === null) {
+      submit.innerHTML = 'Create Task';
+    } else {
+      submit.innerHTML = 'Save Changes';
+    }
+
+    return form.appendChild(submit);
+  };
+
+  // render todo info on click
+
+  const toDoInfo = (toDo, project) => {
+    const div = document.getElementById(`todo-div-${toDo.title.replace(/\s/g, '')}`);
+
+    const testElem = document.getElementsByClassName(
+      `${toDo.title.replace(/\s/g, '')}`,
+    );
+    if (testElem.length > 0) {
+      [...testElem].forEach((elem) => {
+        elem.parentNode.removeChild(elem);
+      });
+      return false;
+    }
+
+    const descriptionPar = document.createElement('p');
+    descriptionPar.innerHTML = `${toDo.description}`;
+    descriptionPar.classList.add(`${toDo.title.replace(/\s/g, '')}`);
+
+    const dueDatePar = document.createElement('p');
+    dueDatePar.innerHTML = `<i class="fas fa-calendar"></i> ${toDo.dueDate.toString()}`;
+    dueDatePar.classList.add(`${toDo.title.replace(/\s/g, '')}`);
+
+    const priorityCheck = document.createElement('div');
+    priorityCheck.classList.add(
+      'priority-check-container',
+      `${toDo.title.replace(/\s/g, '')}`,
+    );
+
+    const priorityPar = document.createElement('p');
+    priorityPar.innerHTML = `Priority: ${toDo.priority}`;
+    priorityPar.classList.add(`${toDo.title.replace(/\s/g, '')}`);
+
+    const checkPar = document.createElement('p');
+    if (toDo.check === true) {
+      checkPar.innerHTML = 'Task Completed';
+    } else {
+      checkPar.innerHTML = 'Pending Task';
+    }
+
+    checkPar.classList.add(`${toDo.title.replace(/\s/g, '')}`);
+
+    priorityCheck.append(priorityPar, checkPar);
+
+    const todoButtons = document.createElement('div');
+    todoButtons.classList.add('todoButtons');
+
+    const deleteButton = document.createElement('button');
+    deleteButton.innerHTML = 'Delete Task';
+    deleteButton.classList.add(`${toDo.title.replace(/\s/g, '')}`);
+    deleteButton.onclick = () => {
+      project.removeToDo(toDo);
+      project.save(projects);
+      window.location.reload();
+    };
+
+    const editButton = document.createElement('button');
+    editButton.innerHTML = 'Edit Task';
+    editButton.classList.add(`${toDo.title.replace(/\s/g, '')}`);
+
+    editButton.onclick = () => {
+      renderForm(project, toDo);
+    };
```

Comment:
Let's prefilled the form with the data already inputted instead of being an empty new form.

Reply:
Done! Thank you for pointing it out, when creating the form we just filled it out with the values of the current To Do.